retroshare-gui/src/gui/StartDialog.cpp

```diff
@@ -90,6 +91,9 @@ StartDialog::StartDialog(QWidget *parent)
 	}
 
 	ui.password_input->setFocus();
+
+	if (Settings->getSheetName() == ":Standard_Dark")
+		ui.rsTopLogo->setPixmap(FilesDefs::getPixmapFromQtResourcePath(":/images/logo/logo_web_nobackground_black.png"));

```

Comment:
Isn't this something you should better do in the qss file? It's weird to hack an image here depending on the specific style RS is using.